examples/web-ui/material/menus/menuMobile.smash

```diff
@@ -0,0 +1,12 @@
+* Set Mobile
+    * Click first level page button {{page}}
```

Comment:
The click functions should be defined under the pages that support them. So, for example...
```
* On Mobile
    * On Layout page {
        // set props
    }
        * Click that big old button
            // etc.
```